llvm/lib/Transforms/Yk/BasicBlockTracer.cpp

```diff
@@ -42,6 +43,12 @@ struct YkBasicBlockTracer : public ModulePass {
     uint32_t FunctionIndex = 0;
     for (auto &F : M) {
       uint32_t BlockIndex = 0;
+      // FIXME: This cause the ykjit to crash when executing compiled
+      // trace at: https://github.com/ykjit/yk/blob/master/ykrt/src/mt.rs#L366
+      // Skip cloned functions
+      if (F.getName().startswith(YK_CLONE_PREFIX)) {
```

Comment:
About this comment. Isn't this what we want anyway? Assuming the cloned functions are the optimised versions, then we don't want them to be traceable anyways?

Reply:
Sorry, that's actually outdated comment. Will remove!

Reply:
Removed :point_right: c98ba8597b38fcfe9bffb9c6d8c5f4e90f6ec466